drafts/inlinemacros.md

```diff
@@ -62,16 +65,19 @@ is equivalent to
 
 ## Inline Reductions
 
-The compiler will do the following rewritings.
+The compiler will do the following rewritings when encountering certain patterns of code
+outside the bodies of inline values and inline methods.
+When these patterns of code appear inside the aforementioned constructs, 
+they will be ignored by the inliner and processed by the compiler as usual.
```

Comment:
About the overloading thing, let's discuss this at https://github.com/odersky/sips/pull/3/files#r29100225.